tests-web/www/js/main.js

```diff
@@ -251,3 +309,48 @@ const useMap = (map) => {
   map.openTest = makeOpenTest(map);
   console.info("New map created! File drops enabled.", container);
 };
+
+// TODO A Leaflet control isn't the right abstraction. We want a popup modal
```

Comment:
As the TODO says. For now, this is usable, but the style is bad and the code is tedious. I'll look into improvements post-conference